scheduler/cloudformation/cloudformation.go

```diff
@@ -268,11 +271,129 @@ func (s *Scheduler) submit(ctx context.Context, tx *sql.Tx, app *scheduler.App,
 		if err := <-done; err != nil {
 			return err
 		}
+		done = make(chan error, 1)
+		s.waitForDeploymentToStabilize(ctx, stackName, ss, done)
```

Comment:
yea, i wish waiter returned the result too so we didn't have to make a duplicate request